transform/mattermost-analytics/models/marts/marketing/fct_security_newsletter.sql

```diff
@@ -0,0 +1,38 @@
+with marketing as (
+    select marketing_id
+        , email as email
+        , server_id
+        , created_at
+        , updated_at
+    from
+        {{ ref('stg_cws__marketing')}}
+    where subscribed_content = 'security_newsletter'
+    qualify row_number() over (
+            partition by email
+            order by
+                created_at desc
+        ) = 1
+),
+security_newsletter as (
+    select distinct m.marketing_id as marketing_id
+        , m.email as email
+        , m.server_id as server_id
+        , l.lead_id as lead_id
+        , cm.campaign_member_id as campaign_member_id
+        , l.lead_id is not null as is_existing_lead
+        , cm.campaign_member_id is not null as is_existing_campaign_member
```

Comment:
I noticed that other similar syncs also require a campaign member status. Is this required for this sync as well?

Reply:
Its required for this sync as well, as if the campaign member exists, we need to associate this campaign to them.

Reply:
Then it should also be added.

Reply:
I added this in the below line